demo/index.html

```diff
@@ -21,14 +21,12 @@
 
   <link rel="import" href="../../paper-item/paper-item.html">
   <link rel="import" href="../paper-menu.html">
-
-  <link rel="stylesheet" href="../../paper-styles/demo.css">
+  <link rel="import" href="../../paper-styles/paper-styles.html">
+  <link rel="import" href="../../paper-styles/demo-pages.html">
 
   <style>
-    .list {
-      display: inline-block;
-      border: 1px solid #ccc;
-      padding: 8px 0;
+    .horizontal-section {
```

Comment:
Is this class also useful for other demos? If so, please move to `paper-styles`.